commit 946c1e0d086cb81c7748eaa3bc21535a0c0dc5ca
Author: Damien Ciabrini <damien.ciabrini@gmail.com>
Date:   Tue Jan 12 01:40:10 2010 +0100

    xkeyboard-config: PC compatibility option for Apple Aluminium Keyboards
    
    Remap some keys to make Aluminium Keyboard behave more like a PC keyboard
      . Print_Screen, Scroll_Lock and Pause instead of F13,F14,F15
      . Numlock instead of Clear
    
    Signed-off-by: Damien Ciabrini <damien.ciabrini@gmail.com>

diff --git a/rules/base.o_s.part b/rules/base.o_s.part
index 2e13cb6..06ee452 100644
--- a/rules/base.o_s.part
+++ b/rules/base.o_s.part
@@ -109,5 +109,6 @@
   esperanto:qwerty      =	+epo(qwerty)
   esperanto:dvorak      =	+epo(dvorak)
   terminate:ctrl_alt_bksp =	+terminate(ctrl_alt_bksp)
+  apple:alupckeys	=	+macintosh_vndr/apple(alupckeys)
 
 
diff --git a/rules/base.xml.in b/rules/base.xml.in
index aca6dc9..ea6132c 100644
--- a/rules/base.xml.in
+++ b/rules/base.xml.in
@@ -5126,6 +5126,12 @@
       </option>
       <option>
         <configItem>
+          <name>apple:alupckeys</name>
+          <description>Apple Aluminium Keyboard: emulate PC keys (Print, Scroll_Lock, Pause, NumLock)</description>
+        </configItem>
+      </option>
+      <option>
+        <configItem>
           <name>shift:breaks_caps</name>
           <_description>Shift cancels CapsLock</_description>
         </configItem>
diff --git a/symbols/macintosh_vndr/apple b/symbols/macintosh_vndr/apple
index e6ab89f..c647ace 100644
--- a/symbols/macintosh_vndr/apple
+++ b/symbols/macintosh_vndr/apple
@@ -125,3 +125,18 @@ xkb_symbols "alukbd" {
       symbols[Group1]= [ NoSymbol, XF86AudioRaiseVolume, NoSymbol, XF86_Switch_VT_12 ]
     };
 };
+
+// Aluminium Keyboard: make F13, F14, F15 PC-ish (Print, Scroll_Lock, Pause)
+partial function_keys keypad_keys
+xkb_symbols "alupckeys" {
+    key <FK13> {
+      type= "PC_ALT_LEVEL2",
+      symbols[Group1]= [ Print, Sys_Req ]
+    };
+    key <FK14> { [ Scroll_Lock ] };
+    key <FK15> {
+      type= "PC_CONTROL_LEVEL2",
+      symbols[Group1]= [ Pause, Break ]
+    };
+    override key <NMLK> { [ Num_Lock, Pointer_EnableKeys ] };
+};
